code/script.js

```diff
@@ -0,0 +1,183 @@
+        <h1 class='course-name'>Technigo Web Development Class of Jan '22 student</h1>
+        `
+}
+
+// Function to filter out all repos forked from Technigo
+const getTechnigoRepos = (data) => {
+    
+    // variable to store the filtered repos
+    const forkedProjects = data.filter(repo => repo.name.includes('project-'))
+    console.log('forked projects:', forkedProjects)
+
+    // Invoking the drawChart function
+    drawChart(forkedProjects.length)
+
+    // Looping through the filtered repos
+    forkedProjects.forEach(repo => {
+
+    // storing reponame for each repo
+    let reponame = repo.name
+    reponame = reponame[0].toUpperCase() + reponame.substring(1)
+
+    //storing the latest update for each repo
+    let recentUpdate = new Date(repo.pushed_at).toLocaleDateString('en', {day: 'numeric', month: 'short', year: 'numeric'})
+
+    // storing the default branch for each repo
+    let defaultBranch = repo.default_branch
+
+    // storing the link for each repo
+    let linkToRepo = repo.html_url
+
+    // Printing the info about projects
+    projectBoard.innerHTML += `
+    <div class='repo-container'>
+        <h2 class='repo-title'>${reponame}</h2>
+        <p>Default branch: ${defaultBranch}</p>
+        <p id='commit-${repo.name}'></p>
+        <p>Most recent push: ${recentUpdate}</p>
+        <button class='repo-link-button'><a class='nav-item' href='${repo.html_url}' target="-blank">Click here to view repo</a></button>
+    `
+    
+//-------------------------------FETCH 2------------------------------------------------
+        fetch(`https://api.github.com/repos/Technigo/${reponame}/pulls?per_page=100`, options)
+            .then(res => res.json())
+            .then(data => {
+
+                // filtering own pull requests
+                const ownPullRequests = data.filter(repo => repo.user.login === username)
+
+                // filtering pull requests done by a team member
+                const otherPullRequests = data.filter(item => item.user.login === 'kolkri' && item.base.repo.name === 'project-weather-app')
+
+                // combining the created arrays
+                const combinedPullRequests = ownPullRequests.concat(otherPullRequests)
+
+                    // looping through own pull requests
+                    combinedPullRequests.forEach(repo => {
+
+                        // storing the reponame of each pull request
+                        reponame = repo.base.repo.name
+
+                        // storing the url for commits for each repo
+                        const commitsForRepos = repo.commits_url
+
+                        // Invoking the fetchCommits function
+                        fetchCommits(commitsForRepos, reponame)
+
+                        // storing the url for comments for each repo -> commented out because not used yet in code
+                        //const commentsForRepos = repo.comments_url
+
```

Comment:
Try to remember to remove the parts you're not going to use before you hand in your project

Reply:
That's a good point 👍  I tried to explain in the comments that this part is not used so the reader doesn't look for it in the features in vain. But I could have explained my reasoning a little more. These parts of code I left as comments because my plan is to use this info when I would have time to work more on this project, let's see if/when that would be possible 🙈

Reply:
That is really smart to plan on going back when you have the time, I like that!